server/public/model/outgoing_oauth_connection.go

```diff
@@ -157,3 +160,32 @@ func (oaf *OutgoingOAuthConnectionGetConnectionsFilter) SetDefaults() {
 		oaf.Limit = defaultGetConnectionsLimit
 	}
 }
+
+// ToURLValues converts the filter to url.Values
+func (oaf *OutgoingOAuthConnectionGetConnectionsFilter) ToURLValues() url.Values {
+	v := url.Values{}
+
+	if oaf.Limit > 0 {
+		v.Set("limit", fmt.Sprintf("%d", oaf.Limit))
+	}
+
+	if oaf.OffsetId != "" {
+		v.Set("offset_id", oaf.OffsetId)
+	}
+
+	if oaf.Audience != "" {
+		v.Set("audience", oaf.Audience)
+	}
+	return v
+}
+
+// OutgoingOAuthConnectionToken is used to return the token for an outgoing connection oauth
+// authentication request
+type OutgoingOAuthConnectionToken struct {
+	AccessToken string
+	TokenType   string
+}
+
+func (ooct *OutgoingOAuthConnectionToken) AsHeaderValue() string {
+	return ooct.TokenType + " " + ooct.AccessToken
```

Comment:
How/when is the `TokenType` defined?

Reply:
Token type is supposed to come with the IDP response: https://www.oauth.com/oauth2-servers/access-tokens/access-token-response/